File: app/src/main/java/com/scujcc/dahuo/function/WalletActivity.java
package com.scujcc.dahuo.function;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.view.ViewGroup;

import com.scujcc.dahuo.R;

public class WalletActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTitle("我的钱包");
        setContentView(R.layout.user_activity_wallet);
    }

    public class WalletAdapter extends RecyclerView.Adapter<WalletAdapter.IndexHolder> {

        /**
         * 此部分待优化, 暂时无解
         */
        public class IndexHolder extends RecyclerView.ViewHolder {
            public IndexHolder(View itemView) {
                super(itemView);
            }
        }

        public class OneHolder extends IndexHolder {

            public OneHolder(View itemView) {
                super(itemView);
            }
        }

        public class TwoHolder extends IndexHolder {

            public TwoHolder(View itemView) {
                super(itemView);
            }
        }

        public class ThreeHolder extends IndexHolder {

            public ThreeHolder(View itemView) {
                super(itemView);
            }
        }



        @Override
        public WalletAdapter.IndexHolder onCreateViewHolder(ViewGroup parent, int viewType) {
            return null;
        }

        @Override
        public void onBindViewHolder(WalletAdapter.IndexHolder holder, int position) {

        }

        @Override
        public int getItemCount() {
            return 0;
        }


    }
}

File: app/src/main/java/com/scujcc/dada/function/WalletActivity.kt
package com.scujcc.dada.function

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Toast

import com.scujcc.dada.R
import kotlinx.android.synthetic.main.user_activity_wallet.*

class WalletActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        title = "我的钱包"
        setContentView(R.layout.user_activity_wallet)
        buttonClick()
    }

    private fun buttonClick() {
        recharge_button.setOnClickListener { Toast.makeText(applicationContext, "充值", Toast.LENGTH_SHORT).show() }
        detail_button.setOnClickListener { Toast.makeText(applicationContext, "充值详情", Toast.LENGTH_SHORT).show() }
        withdraw_button.setOnClickListener { Toast.makeText(applicationContext, "提现", Toast.LENGTH_SHORT).show() }
        money_detail_button.setOnClickListener { Toast.makeText(applicationContext, "提现明细", Toast.LENGTH_SHORT).show() }
        coupon_detail_button.setOnClickListener { Toast.makeText(applicationContext, "优惠券详情", Toast.LENGTH_SHORT).show() }
        coupon_button.setOnClickListener { Toast.makeText(applicationContext, "使用记录", Toast.LENGTH_SHORT).show() }
    }
}
